requirements-test.txt

```diff
@@ -1,24 +1,33 @@
 certifi==2016.8.31 \
-#    --hash=sha256:f7708a42d86f29ccc7c8c4ff9d34a8d854d8d78eb2973d1f28406bb43d6b8919
+    --hash=sha256:172844a3e2896c924e6aa2657b2e56586a724f27edd0e9592be9353f3a3f6c73
```

Comment:
It looks like the hash differences are because some packages have multiple formats. Eg, certifi is available as a .whl and a .tar.gz on https://pypi.python.org/pypi?:action=show_md5&digest=0931a7d2576083826efa419d695117a3:
```
➜  sha256sum *
172844a3e2896c924e6aa2657b2e56586a724f27edd0e9592be9353f3a3f6c73  certifi-2016.8.31-py2.py3-none-any.whl
f7708a42d86f29ccc7c8c4ff9d34a8d854d8d78eb2973d1f28406bb43d6b8919  certifi-2016.8.31.tar.gz
```

I know it's going to be a hassle, but can you make sure all hashes from each package are specified?